### PR TITLE
Glide updates for dynamic haproxy reload

### PR DESCRIPTION
**Pull request by ironcladlou**
Add prerequisite dependencies to support #19073. This is done separately as a special case to accelerate completion of the feature by reducing the need to manage unrelated Origin dependency changes.

Replaces https://github.com/openshift/origin/pull/19936
/cc @deads2k @ramr

### CODE REVIEW

**Comment by ironcladlou** [comment]
test /cc @openshift/sig-network-edge

File: vendor/k8s.io/kubernetes/pkg/kubectl/genericclioptions/print_flags.go [hunk 1]
```diff
@@ -97,7 +97,7 @@ func (f *PrintFlags) AddFlags(cmd *cobra.Command) {
 	f.NamePrintFlags.AddFlags(cmd)
 
 	if f.OutputFormat != nil {
-		cmd.Flags().StringVarP(f.OutputFormat, "output", "o", *f.OutputFormat, fmt.Sprintf("Output format. One of: %s.", strings.Join(f.AllowedFormats(), "|")))
+		cmd.Flags().StringVarP(f.OutputFormat, "output", "o", *f.OutputFormat, "Output format. One of: json|yaml|wide|name|custom-columns=...|custom-columns-file=...|go-template=...|go-template-file=...|jsonpath=...|jsonpath-file=... See custom columns [http://kubernetes.io/docs/user-guide/kubectl-overview/#custom-columns], golang template [http://golang.org/pkg/text/template/#pkg-overview] and jsonpath template [http://kubernetes.io/docs/user-guide/jsonpath].")
```
Comment (deads2k) [inline vendor/k8s.io/kubernetes/pkg/kubectl/genericclioptions/print_flags.go]: @ironcladlou you need to re-run.  This is backleveling us.  The change you need made it into the branch a few minutes ago.

**Comment by ironcladlou** [comment]
@deads2k picked up the latest kube deps from master, PTAL

**Comment by deads2k** [comment]
@ironcladlou you need to regenerate code.

**Comment by deads2k** [comment]
still failing verify

**Comment by deads2k** [comment]
/lgtm

but verify fails

**Comment by ramr** [comment]
Closing this out as per an email exchange with @deads2k  - changes are all pulled into PR #19073

**Pull request closed without merging**